docs/packages/pkg/Boost.rst

```diff
@@ -88,6 +88,52 @@ config file (``boost/config/user.hpp``):
     #define BOOST_MPL_CFG_NO_PREPROCESSED_HEADERS
     #define BOOST_MPL_LIMIT_LIST_SIZE 3
 
+- Option ``REQUESTED_PYTHON_VERSION=<VALUE>``, if the required components of the python library
+  arguments are located, will result in ``user_config.jam`` being appended with the following line:
+
+.. code-block:: none
+
+  "using python  : <requested_version_number> : <path to python executable> :
+  <path to python include directory> : <path to directory containing the python library> ;\n".
+
+  This allows choosing a specific python version to build the Boost Python library against.
+  Example:
+
+.. code-block:: cmake
+
+    hunter_config(
+      Boost
+      VERSION ${HUNTER_Boost_VERSION}
+      CMAKE_ARGS
+      REQUESTED_PYTHON_VERSION=3.6.7
+    )
+    #FindBoost python<x> components after boost version 1.67 require version suffix
```

Comment:
This part is big and is not directly about CMAKE_ARGS, I think it make sense to add subsection with separate Python 2 and Python 3 examples.

Reply:
Ok, will do

Reply:
I added a subsection and split up the CMake into two examples, one for Python 2 and one for Python 3. I've put the note about why the version check is needed between the two examples. Seems to look fine after docs are built locally.